.github/release-drafter.yml

```diff
@@ -40,7 +40,6 @@ template: |
   - `docker.elastic.co/beats-dev/golang-crossbuild:$RESOLVED_VERSION-arm-debian12` - linux/armv5, linux/armv6
   - `docker.elastic.co/beats-dev/golang-crossbuild:$RESOLVED_VERSION-armel` - linux/armv5, linux/armv6
   - `docker.elastic.co/beats-dev/golang-crossbuild:$RESOLVED_VERSION-armel-debian9` - linux/armv5, linux/armv6
```

Comment:
To stay consistent, it seems like we should also drop armel for debian9 and update the associated README line too.